api_config.yaml

```diff
@@ -0,0 +1,14 @@
+type: google.api.Service
+config_version: 3
+
+#
+# Name of the service configuration.
+#
+name: testbed.endpoints.MY_PROJECT_ID.cloud.goog
```

Comment:
If you expect users to replace `MY_PROJECT_ID` to an appropriate value, please at least add a `NOTE` comment explaining the expected usage.